module/canvas/ability-template.mjs

```diff
@@ -48,6 +48,7 @@ export default class AbilityTemplate extends MeasuredTemplate {
       fillColor: game.user.color,
       flags: { dnd5e: { origin: item.uuid } }
     };
+    if ( item.type === "spell" ) foundry.utils.mergeObject(templateData.flags, {"dnd5e.spellLevel": item.system.level});
```

Comment:
`mergeObject` might be a bit overkill here, since `flags.dnd5e` always exists. 🤔

Reply:
I just followed the same pattern already established here:
https://github.com/foundryvtt/dnd5e/blob/bdc76e271d1ebae7092824c4f0e7c5299d7172e4/module/documents/item.mjs#L842